### PR TITLE
Updated my profile

### PR DESCRIPTION
**Pull request by Jaideep25-tech**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4788"></a>

### CODE REVIEW

File: data/Jaideep25-tech.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "Jaideep Singh",
   "type": "personal",
+  "displayStatsPublic": true,
   "bio": "Full Stack **Flutter** developer | Open Source Advocate | Freelancer | Mentor",
   "tags": [
```
Comment (priyankarpal) [inline data/Jaideep25-tech.json:6]: One suggestion please remove the tag `"Full Stack Cross Platform Mobile Application Development",` and add another.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/Jaideep25-tech.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "Jaideep Singh",
   "type": "personal",
+  "displayStatsPublic": true,
```
[the remark this answers is not in the document]
Comment (Jaideep25-tech) [reply data/Jaideep25-tech.json]: now I have removed it. Can now it be merged?

**Comment by Jaideep25-tech** [comment]
@priyankarpal i have made the changes accordingly can you please now proceed with merging it?

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**